### PR TITLE
German Translation of 3.22.0-PTU

### PR DESCRIPTION
**Pull request by ROBdk97**
Re-submitted so it could be reviewed by @Auhrus.
(I can merge myself)

### CODE REVIEW

File: data/Localization/german_(germany)/global.ini [hunk 1]
```diff
@@ -8513,8 +8560,8 @@ Hints_Melee_KnifeLight=Um einen schnellen Hieb auszuführen, tippe auf ~action(p
 Hints_Melee_LightAttacks=Um einen schnellen Jab zu werfen, tippe auf ~action(player|melee_AttackLightLeft) oder ~action(player|melee_AttackLightRight).<left><right></right></left>
 Hints_Melee_Stun=Wenn du von Fäusten getroffen wirst, erleidest du Betäubungsschaden. Dieser ist nicht tödlich, aber wenn du zu oft getroffen wirst, wirst du niedergeschlagen.
 Hints_Melee_TakedownFists=Drücken Sie ~action(player|weapon_melee) für einen nicht-tödlichen Takedown.
-Hints_Melee_TakedownGun=Press ~action(player|weapon_melee) <takedown gun="">for a non-lethal takedown.</takedown>
-Hints_Melee_TakedownWeapon=Press ~action(player|weapon_melee)<takedown melee="">&nbsp;for a lethal takedown.</takedown>
+Hints_Melee_TakedownGun=Drücken Sie ~action(player|weapon_melee) <takedown gun="">für eine nicht-tödliche Überwältigung.</takedown>
+Hints_Melee_TakedownWeapon=Drücken Sie ~action(player|weapon_melee)<takedown melee="">&nbsp;für eine tödliche Überwältigung.</takedown>
```
Comment (Auhrus) [inline data/Localization/german_(germany)/global.ini:8564]: I would replace the "Drücken Sie" with a "Drücke" (and the line above it)
and maybe find another word for "Überwältigung". I would maybe just leave it by "takedown"

File: data/Localization/german_(germany)/global.ini [hunk 2]
```diff
@@ -51778,7 +51987,7 @@ item_NameDSTC_ARGO_Retro_Joint=Retro Schubdüse
 item_NameDSTC_FP_G4GI_S2_Q1=Dragon STC Blau Haupt Schubdüse
 item_NameDSTC_FP_G4GI_S3_Q1=Dragon STC Rot Hauptmotor
 item_NameDSTC_FP_G4GI_S4_Q1=Dragon STC Silber Hauptmotor
-item_NameDebugGun=<-=FEHLT=->
+item_NameDebugGun=<-=MISSING=->
```
Comment (Auhrus) [inline data/Localization/german_(germany)/global.ini:51990]: Reverse the "MISSING" here to "FEHLT" or similar
The same applies to the lines:
[52548](https://github.com/Dymerz/StarCitizen-Localization/pull/196/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R52548)
[53014](https://github.com/Dymerz/StarCitizen-Localization/pull/196/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R53014)
[53154](https://github.com/Dymerz/StarCitizen-Localization/pull/196/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R53154)
[53386](https://github.com/Dymerz/StarCitizen-Localization/pull/196/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R53386)
[53358](https://github.com/Dymerz/StarCitizen-Localization/pull/196/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R53358)

**Review by Auhrus — CHANGES_REQUESTED** [review]
Only suggestions 😉

File: data/Localization/german_(germany)/global.ini (hunk 2 above)
Comment (ROBdk97) [reply data/Localization/german_(germany)/global.ini:51990]: I would prefer MISSING because it's not translated text

**Pull request merged**